tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -116,7 +116,7 @@ def _map_status_name(self, status):
 
     def _get_expected_status_from_data(self, actual_status, data):
         """
-        Gets the expected status from a |data| dictionary.
+        Gets the expected statuses from a |data| dictionary.
 
         If there is no expected status in data, the actual status is returned.
```

Comment:
```suggestion
        If there is no expected statuses in data, the actual status is returned.
```

Reply:
I think this block is still grammatically correct since we're talking about a single-expected-status case, and the following paragraph explains the multi-status part.